charts/console/templates/softserve.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.gitServer }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: softserve
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: softserve
```

Comment:
I actually think this might goof up now that you changed the labels in template.metadata.labels to be different from `app: softserve`.  Also the deployment object itself should have the default lables like in other ones.